ruby-challenges.rb

```diff
@@ -33,3 +67,43 @@
 # Expected output example: my_bike.pedal_faster(18) => 28
 # Expected output example: my_bike.brake(5) => 23
 # Expected output example: my_bike.brake(25) => 0
+
+class Bike
+    def initialize(model)
+        @model = model
+        @wheels = 2
+        @current_speed = 0
+    end
+
+    def get_bike_info
+        "The Bike is a #{model}, has #{wheels} wheels, and is going #{current_speed}"
```

Comment:
Don't forget that you are accessing instance variables here, so you will want @model, @wheels, and @current_speed.